### PR TITLE
Feature/70453 empty column

### PR DESCRIPTION
**Pull request by Camwyn**
🎫 https://central.tri.be/issues/70453
Fills the empty column with the user name and email.

Also fixes an `unidentified variable` warning in RSVP.php

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:x: | There must be a single space after the first parenthesis | [Attendees.php:592](https://github.com/moderntribe/event-tickets/blob/406f2ceb618d4e6b725677c07bfda19e0b2f1df6/src/Tribe/Attendees.php#L592)
:x: | There must be a single space before the last parenthesis | [Attendees.php:592](https://github.com/moderntribe/event-tickets/blob/406f2ceb618d4e6b725677c07bfda19e0b2f1df6/src/Tribe/Attendees.php#L592)
:x: | Don't leave error_log lying around | [Attendees.php:597](https://github.com/moderntribe/event-tickets/blob/406f2ceb618d4e6b725677c07bfda19e0b2f1df6/src/Tribe/Attendees.php#L597)
:x: | Comma required after last value in array declaration | [Attendees.php:603](https://github.com/moderntribe/event-tickets/blob/406f2ceb618d4e6b725677c07bfda19e0b2f1df6/src/Tribe/Attendees.php#L603)
:x: | There must be a single space after the first square bracket | [Attendees.php:612](https://github.com/moderntribe/event-tickets/blob/406f2ceb618d4e6b725677c07bfda19e0b2f1df6/src/Tribe/Attendees.php#L612)
:x: | There must be a single space before the last square bracket | [Attendees.php:612](https://github.com/moderntribe/event-tickets/blob/406f2ceb618d4e6b725677c07bfda19e0b2f1df6/src/Tribe/Attendees.php#L612)
:x: | There must be a single space after the first square bracket | [Attendees.php:613](https://github.com/moderntribe/event-tickets/blob/406f2ceb618d4e6b725677c07bfda19e0b2f1df6/src/Tribe/Attendees.php#L613)
:x: | There must be a single space before the last square bracket | [Attendees.php:613](https://github.com/moderntribe/event-tickets/blob/406f2ceb618d4e6b725677c07bfda19e0b2f1df6/src/Tribe/Attendees.php#L613)
:x: | There must be a single space after the first square bracket | [Attendees.php:613](https://github.com/moderntribe/event-tickets/blob/406f2ceb618d4e6b725677c07bfda19e0b2f1df6/src/Tribe/Attendees.php#L613)
:x: | There must be a single space before the last square bracket | [Attendees.php:613](https://github.com/moderntribe/event-tickets/blob/406f2ceb618d4e6b725677c07bfda19e0b2f1df6/src/Tribe/Attendees.php#L613)
:x: | There must be a single space after the first square bracket | [Attendees.php:614](https://github.com/moderntribe/event-tickets/blob/406f2ceb618d4e6b725677c07bfda19e0b2f1df6/src/Tribe/Attendees.php#L614)
:x: | There must be a single space before the last square bracket | [Attendees.php:614](https://github.com/moderntribe/event-tickets/blob/406f2ceb618d4e6b725677c07bfda19e0b2f1df6/src/Tribe/Attendees.php#L614)
:x: | There must be a single space after the first square bracket | [Attendees.php:614](https://github.com/moderntribe/event-tickets/blob/406f2ceb618d4e6b725677c07bfda19e0b2f1df6/src/Tribe/Attendees.php#L614)
:x: | There must be a single space before the last square bracket | [Attendees.php:614](https://github.com/moderntribe/event-tickets/blob/406f2ceb618d4e6b725677c07bfda19e0b2f1df6/src/Tribe/Attendees.php#L614)
:x: | Don't leave error_log lying around | [Attendees.php:615](https://github.com/moderntribe/event-tickets/blob/406f2ceb618d4e6b725677c07bfda19e0b2f1df6/src/Tribe/Attendees.php#L615)
:x: | There must be a single space after the first parenthesis | [Attendees.php:615](https://github.com/moderntribe/event-tickets/blob/406f2ceb618d4e6b725677c07bfda19e0b2f1df6/src/Tribe/Attendees.php#L615)
:x: | There must be a single space before the last parenthesis | [Attendees.php:615](https://github.com/moderntribe/event-tickets/blob/406f2ceb618d4e6b725677c07bfda19e0b2f1df6/src/Tribe/Attendees.php#L615)
:x: | There must be a single space after the first square bracket | [Attendees.php:616](https://github.com/moderntribe/event-tickets/blob/406f2ceb618d4e6b725677c07bfda19e0b2f1df6/src/Tribe/Attendees.php#L616)
:x: | There must be a single space before the last square bracket | [Attendees.php:616](https://github.com/moderntribe/event-tickets/blob/406f2ceb618d4e6b725677c07bfda19e0b2f1df6/src/Tribe/Attendees.php#L616)
:x: | Don't leave error_log lying around | [Attendees.php:617](https://github.com/moderntribe/event-tickets/blob/406f2ceb618d4e6b725677c07bfda19e0b2f1df6/src/Tribe/Attendees.php#L617)
:x: | There must be a single space after the first parenthesis | [Attendees.php:617](https://github.com/moderntribe/event-tickets/blob/406f2ceb618d4e6b725677c07bfda19e0b2f1df6/src/Tribe/Attendees.php#L617)
:x: | There must be a single space before the last parenthesis | [Attendees.php:617](https://github.com/moderntribe/event-tickets/blob/406f2ceb618d4e6b725677c07bfda19e0b2f1df6/src/Tribe/Attendees.php#L617)
:x: | There must be a single space after the first square bracket | [Attendees.php:617](https://github.com/moderntribe/event-tickets/blob/406f2ceb618d4e6b725677c07bfda19e0b2f1df6/src/Tribe/Attendees.php#L617)
:x: | There must be a single space before the last square bracket | [Attendees.php:617](https://github.com/moderntribe/event-tickets/blob/406f2ceb618d4e6b725677c07bfda19e0b2f1df6/src/Tribe/Attendees.php#L617)

> via jenkins/codesniffer-phpcs

File: src/Tribe/RSVP.php [hunk 1]
```diff
@@ -1046,6 +1046,8 @@ public function front_end_tickets_form( $content ) {
 			}
 		}
 
+		$must_login = ! is_user_logged_in() && $this->login_required();
```
Comment (Camwyn) [inline src/Tribe/RSVP.php]: @bordoni I requested you for this - this was being defined after it was being passed to the filter on line 1071

File: src/Tribe/Attendees.php [hunk 2]
```diff
@@ -588,8 +588,35 @@ public function maybe_generate_csv() {
 			// create a file pointer connected to the output stream
 			$output = fopen( 'php://output', 'w' );
 
+			// Get indexes by keys
+			$flip  = array_flip($items[0]);
+			$prime = $flip['Primary Information'];
+			$name  = $flip['Customer Name'];
+			$email = $flip['Customer Email Address'];
+
+			error_log(
```
Comment (mitogh) [inline src/Tribe/Attendees.php]: I think this one should be removed. `error_log`

Comment (mitogh) [reply src/Tribe/Attendees.php]: Same as other instances of `error_log`

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:x: | There must be a single space after the first parenthesis | [Attendees.php:592](https://github.com/moderntribe/event-tickets/blob/d9ad05ef533af90c5ce5ae23f8c449a1b4e8b56d/src/Tribe/Attendees.php#L592)
:x: | There must be a single space before the last parenthesis | [Attendees.php:592](https://github.com/moderntribe/event-tickets/blob/d9ad05ef533af90c5ce5ae23f8c449a1b4e8b56d/src/Tribe/Attendees.php#L592)
:x: | There must be a single space after the first square bracket | [Attendees.php:599](https://github.com/moderntribe/event-tickets/blob/d9ad05ef533af90c5ce5ae23f8c449a1b4e8b56d/src/Tribe/Attendees.php#L599)
:x: | There must be a single space before the last square bracket | [Attendees.php:599](https://github.com/moderntribe/event-tickets/blob/d9ad05ef533af90c5ce5ae23f8c449a1b4e8b56d/src/Tribe/Attendees.php#L599)
:x: | There must be a single space after the first square bracket | [Attendees.php:600](https://github.com/moderntribe/event-tickets/blob/d9ad05ef533af90c5ce5ae23f8c449a1b4e8b56d/src/Tribe/Attendees.php#L600)
:x: | There must be a single space before the last square bracket | [Attendees.php:600](https://github.com/moderntribe/event-tickets/blob/d9ad05ef533af90c5ce5ae23f8c449a1b4e8b56d/src/Tribe/Attendees.php#L600)
:x: | There must be a single space after the first square bracket | [Attendees.php:600](https://github.com/moderntribe/event-tickets/blob/d9ad05ef533af90c5ce5ae23f8c449a1b4e8b56d/src/Tribe/Attendees.php#L600)
:x: | There must be a single space before the last square bracket | [Attendees.php:600](https://github.com/moderntribe/event-tickets/blob/d9ad05ef533af90c5ce5ae23f8c449a1b4e8b56d/src/Tribe/Attendees.php#L600)
:x: | There must be a single space after the first square bracket | [Attendees.php:601](https://github.com/moderntribe/event-tickets/blob/d9ad05ef533af90c5ce5ae23f8c449a1b4e8b56d/src/Tribe/Attendees.php#L601)
:x: | There must be a single space before the last square bracket | [Attendees.php:601](https://github.com/moderntribe/event-tickets/blob/d9ad05ef533af90c5ce5ae23f8c449a1b4e8b56d/src/Tribe/Attendees.php#L601)
:x: | There must be a single space after the first square bracket | [Attendees.php:601](https://github.com/moderntribe/event-tickets/blob/d9ad05ef533af90c5ce5ae23f8c449a1b4e8b56d/src/Tribe/Attendees.php#L601)
:x: | There must be a single space before the last square bracket | [Attendees.php:601](https://github.com/moderntribe/event-tickets/blob/d9ad05ef533af90c5ce5ae23f8c449a1b4e8b56d/src/Tribe/Attendees.php#L601)
:x: | There must be a single space after the first square bracket | [Attendees.php:602](https://github.com/moderntribe/event-tickets/blob/d9ad05ef533af90c5ce5ae23f8c449a1b4e8b56d/src/Tribe/Attendees.php#L602)
:x: | There must be a single space before the last square bracket | [Attendees.php:602](https://github.com/moderntribe/event-tickets/blob/d9ad05ef533af90c5ce5ae23f8c449a1b4e8b56d/src/Tribe/Attendees.php#L602)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:x: | There must be a single space after the first parenthesis | [Attendees.php:592](https://github.com/moderntribe/event-tickets/blob/6e3dc3468be1fa673cb8059da05d2428378065f7/src/Tribe/Attendees.php#L592)
:x: | There must be a single space before the last parenthesis | [Attendees.php:592](https://github.com/moderntribe/event-tickets/blob/6e3dc3468be1fa673cb8059da05d2428378065f7/src/Tribe/Attendees.php#L592)

> via jenkins/codesniffer-phpcs

File: src/Tribe/Attendees.php (hunk 2 above)
Comment (Camwyn) [reply src/Tribe/Attendees.php]: dangit...corrected.

**Pull request merged**